mysql-test/suite/tianmu/t/issue1187.test

```diff
@@ -0,0 +1,45 @@
+--source include/have_tianmu.inc
+
+--disable_warnings
+DROP DATABASE IF EXISTS issuse1187_test;
+--enable_warnings
+
+CREATE DATABASE issuse1187_test;
+USE issuse1187_test;
+
+create table t1(id int,name varchar(5)) ;
+insert into t1 values(1,'AAA'),(2,'BBB');
+alter table t1 add column age int null;
+select * from t1;
+
+create table t2(id int,name varchar(5)) ;
+insert into t2 values(1,'AAA'),(2,'BBB');
+alter table t2 add column age int not null;
```

Comment:
if it is convenient，please fix it together，when the type is varchar not null

Reply:
if it is convenient，please fix it together when the type is varchar not null
as below case:
```
create table ttt(id int,name varchar(5));
insert into ttt values(1,'AAA'),(2,'BBB');
alter table ttt add column age varchar(5) not null;
select * from ttt;
```